### PR TITLE
Match info screen

### PR DESCRIPTION
**Pull request by justin212553**
Still working on it, I will tell you when it is over

### CODE REVIEW

File: app/src/main/java/models/Match.java [hunk 1]
```diff
@@ -64,4 +75,37 @@ public int getMatchType()
...
+        user.add("follows", getString("objectID"));
+        user.saveInBackground();
+    }
+
+    @Override
+    public boolean getFollow() {
+        Object a = getCurrentUser().get("follows");
+        if(a == null)
+            return false;
+        return ((ArrayList)a).contains(getString("objectID"));
```
Comment (akp5896) [inline app/src/main/java/models/Match.java:109]: "objectId"

File: app/src/main/java/com/example/sc2infoapp/MatchDetailActivity.java [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+            @Override
+            public void onTabReselected(TabLayout.Tab tab) {}
+        });
+    }
+
+    public void getExternalTeam(String left, String right) {
+        LiquipediaClient client = new LiquipediaClient();
+
+        try {
+            String imagePathLeft = client.getPageByName(left);
```
Comment (akp5896) [inline app/src/main/java/com/example/sc2infoapp/MatchDetailActivity.java:132]: Did you test this, it works?

File: app/src/main/java/fragments/MatchInfoFragment.java [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+                }
+                left = objects.get(0);
+            }
+        });
+
+        return founded;
+    }
+
+    public String lineupIntoString(ArrayList<ParseUser> lineups) {
+        String a = "";
```
Comment (akp5896) [inline app/src/main/java/fragments/MatchInfoFragment.java:126]: Maybe use StringBuilder?

**Comment by akp5896** [comment]
6. getParseTeam - we currently don't have team pictures, but we can add them.
7.  I think replace magical numbers in switch statements with IMatch.EXTERNAL etc. That would make code more readable.

**Review by akp5896 — APPROVED** [review]

**Review by stevens34400 — APPROVED** [review]

**Pull request merged**